package.json

```diff
@@ -104,11 +100,25 @@
         "@popper/core>react-dom": "19",
         "@reach/tabs>react": "19",
         "@reach/tabs>react-dom": "19",
+        "@reach/auto-id>react": "19",
+        "@reach/auto-id>react-dom": "19",
+        "@reach/utils>react": "19",
+        "@reach/utils>react-dom": "19",
+        "@reach/descendants>react": "19",
+        "@reach/descendants>react-dom": "19",
+        "@reach/polymorphic>react": "19",
+        "@reach/polymorphic>react-dom": "19",
         "eslint": "9",
         "react-focus-on>react": "19",
         "react-focus-on>react-dom": "19",
         "react-popper>react": "19",
         "react-popper>react-dom": "19",
+        "react-remove-scroll>react": "19",
+        "react-remove-scroll>react-dom": "19",
+        "react-remove-scroll-bar>react": "19",
+        "react-remove-scroll-bar>react-dom": "19",
+        "react-style-singleton>react": "19",
+        "react-style-singleton>react-dom": "19",
```

Comment:
I had to add a few more transient deps of some of our older packages I want to be explicit here and not do a catch all so we know when we remove these we can also clean up here